compiler/rustc_target/src/spec/apple_sdk_base.rs

```diff
@@ -20,14 +21,15 @@ fn target_cpu(arch: Arch) -> String {
         I386 => "yonah",
         X86_64 => "core2",
         X86_64_macabi => "core2",
+        Arm64_macabi => "apple-a7",
```

Comment:
I'm surprised by this (and the other instances) because the DTK (and presumably the real commercial releases) are `apple-a12` at a minimum.

Reply:
Thanks! I'd overlooked this. I've updated it accordingly to a12